### DISCUSSION TITLE
Documentation for using ApolloClient with NhostApolloProvider in React

### DISCUSSION THREAD

**Discussion by wollerman** [nhost/nhost#2418] [Q&A]
**Is your feature request related to a problem? Please describe.**
ApolloClient has lots of functionality such as managing [InMemoryCache](https://www.apollographql.com/docs/react/api/cache/InMemoryCache) policies to allow for managing specific field policies, which is [especially useful when it comes to pagination](https://www.apollographql.com/docs/react/pagination/core-api/#defining-a-field-policy).

Right now my app structure looks like

```typescript

const nhost = new NhostClient({...});

return (
<NhostProvider nhost={nhost}>
  <NhostApolloProvider nhost={nhost} headers={headers}>
    <Navigation />
  </NhostApolloProvider>
</NhostProvider>
);
```

How can we utilize these features of ApolloClient based on the recommended NhostApolloProvider setup?

https://docs.nhost.io/reference/react/apollo

**Describe the solution you'd like**
Provide documentation on the recommended way to set up ApolloClient while using NhostClient with NhostApolloProvider.

**Describe alternatives you've considered**
I haven't yet; I'm looking for guidance here first. Thank you!

**Comment by dbarrosop** [comment]
Please, don't use github issues for asking questions/guidance. Use discord or github discussions instead. Thanks!

**Reply by wollerman** [reply]
Sounds good; sorry for the noise!
